lib/state/ui/reducer.ts

```diff
@@ -34,6 +34,21 @@ const visiblePanes: A.Reducer<string[]> = (
   return state;
 };
 
+const showNoteInfo: A.Reducer<boolean> = (state = false, action) => {
+  if ('NOTE_INFO_TOGGLE' === action.type) {
+    return action.show;
+
+    // @todo?
+    // originally if note info toggles in,
+    // showNavigation and editingTags were set to false
+    //     showNoteInfo: { $set: true },
+    //     showNavigation: { $set: false },
+    //     editingTags: { $set: false },
+  }
+
```

Comment:
Again, I believe we don't need to handle these cases explicitly, but I hope someone can verify that

Reply:
we can verify it!

Reply:
in testing I can see that it's more of a choice to _change_ the behavior or keep it. if we take this out then the note info stays open when we flip the navigation panel.

Reply:
I haven't been able to make that happen, what are the steps? If I open note info, there's no way to open the navigation panel (any click is an "outside click" that closes the note info).

Reply:
From Slack: "Use the keyboard shortcut Cmd + t and flip the tag drawer/nav panel open and closed"

I tried this and it does seem to close the info panel, so I'm gonna say.. it's fine.

Reply:
sorry. I should have clarified that - obviously this code closes the note info panel when toggling the navigation panel. _however_, when I removed the `case 'App.toggleNavigation'` as it seemed like you were suggesting would be possible, then the note info would stay open on navigation.

That is, this is an answer to the question "can we skip handling these cases explicitly" and that answer is "if we want and decide to."